Package.swift

```diff
@@ -0,0 +1,70 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "j2objc-dynamic-frameworks-swift-package",
+    platforms: [
+        .iOS(.v12)
+    ],
+    products: [
+        .library(
+            name: "JRE_Core",
+            targets: ["JRE_Core"]
+        ),
+        .library(
+            name: "JSR305",
+            targets: ["JSR305"]
+        ),
+        .library(
+            name: "JUnit",
+            targets: ["JUnit"]
+        ),
+        .library(
+            name: "JRE_Concurrent",
+            targets: ["JRE_Concurrent"]
+        ),
+        .library(
+            name: "JSON",
+            targets: ["JSON"]
+        ),
+        .library(
+            name: "JRE_Util",
+            targets: ["JRE_Util"]
+        )
+    ],
+    dependencies: [],
+    targets: [
+        .binaryTarget(
+            name: "JRE_Core",
+            url: "https://sdk-artifactory.justride.com/artifactory/ios-artifacts/JRE_Core/2023.09.26/JRE_Core.xcframework.zip",
+            checksum: "d60f11060d306365af97e98ba8f785259222698f752b5d6dded8f18fc5fabd21"
+        ),
+        .binaryTarget(
+            name: "JSR305",
```

Comment:
You have no dependency relationships defined between these products/targets. I think that's fine though for how we're going to link to these libraries from the Justride SDK. If the absence of the dependency relationships causes a problem down the line, we can deal with it then.

Reply:
Yes, for our purpose it should be okay, but it's definitely sth to consider when problems occur